test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1

```diff
@@ -133,8 +133,13 @@ Describe "Process Parent property" -Tags "CI" {
     It "Has Parent process property in linux" -Skip:(-not $IsLinux) {
         # Added new test to check for bug as described in issue https://github.com/PowerShell/PowerShell/issue/12908
             $powershellexe = (Get-Process -Name "$((Get-Process -Id $PID).Name)").mainmodule.filename
-            & $powershellexe -noprofile -command '(Get-Process -Name "$((Get-Process -Id $PID).Name)").Parent' | Should -Not -BeNullOrEmpty
-        }
+            & $powershellexe -noprofile -command "echo '#!/bin/bash' > 't ( e ( s ) t )'"
+            & $powershellexe -noprofile -command "echo './src/powershell-unix/bin/Debug/net5.0/linux-x64/publish/pwsh' >> 't ( e ( s ) t )'"
```

Comment:
Please use `$pwsh = "$PSHOME/pwsh"` to get current pwsh.

Reply:
It's we're using current pwsh (without building), wouldn't that return null since the bug is still there in it?

Reply:
No, this references pwsh we build and test - Pester runs the test file in the pwsh.